metadata/src/main/java/org/apache/kafka/controller/ReplicationControlManager.java

```diff
@@ -1994,6 +2014,24 @@ private void listReassigningTopic(ListPartitionReassignmentsResponseData respons
             setReplicas(Replicas.toList(partition.replicas)));
     }
 
+    // Make it public to be visible in test.
```

Comment:
nit: we usually write `// Visible for tests`

does it need to be public? Can it be package-private?

Reply:
Cool, done.